PYME/LMVis/pipeline.py

```diff
@@ -705,7 +705,7 @@ def _ds_from_file(self, filename, **kwargs):
                 #old style matlab import
                 ds = tabular.MatfileSource(filename, kwargs['FieldNames'], kwargs['VarName'])
             else:
-                if 'Multichannel' in kwargs.keys():
+                if 'Multichannel' in kwargs.keys() and kwargs.get('Multichannel'):
```

Comment:
Could simplify a little as

```suggestion
                if kwargs.get('Multichannel', False):
```

Not really important here, but generally a better pattern as it should also be faster than `in keys()`, as it's faster to access with a missing key and handle the `KeyError` or to call `get()` with a default than to iterate keys - especially as dictionary sizes increase.

Reply:
I like it. Let's switch to this.